locales/languages.json

```diff
@@ -1 +1 @@
-["en", "zh", "de", "es", "fr", "cs"]
+["en", "ru", "zh", "de", "es", "fr", "cs"]
```

Comment:
Ideally this list should be alphabetical, with the exception of universal en being first.

Reply:
Oops missed you review. Atm they are not alphabetical, but I would enforce this at rendering anyway if we wanted to? The order of the list here is not meaningful